api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-profile-get.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Get profile"
+description: "Read properties and relationships of a profile object."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Get profile
+
+Namespace: Microsoft.Fast.PI.ViewGenerator.V2.Profile.Models
+
+Read properties and relationships of a [profile](../resources/microsoft.fast.pi.viewgenerator.v2.profile.models-profile.md) object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Provide applicable permissions.**|
+|Delegated (personal Microsoft account)|**TODO: Provide applicable permissions.**|
+|Application|**TODO: Provide applicable permissions.**|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /user/{userId}/profile
```

Comment:
@wandeg Should we also be showing the /me alias here?
```GET /me/profile```

Reply:
+1, I think we should.